build.gradle

```diff
@@ -12,9 +13,18 @@ repositories {
 }
 
 group = "io.trocco"
-version = "0.1.0"
 description = "Bigquery output for embulk"
 
+version = {
+    def baseVersion = "0.1.0"
```

Comment:
remove `baseVersion`. use `VersionDetails()` only.

see: https://github.com/palantir/gradle-git-version

Reply:
I got it.